search/tests/convert/granules.spec.js

```diff
@@ -244,14 +246,22 @@ describe('granuleToItem', () => {
           id: 1,
           collection: 10,
           geometry: { type: 'Point', coordinates: [77, 39] },
+          bbox: undefined,
           properties: {
             provider: 'USA',
             datetime: '0',
             start_datetime: '0',
             end_datetime: '1'
           },
           type: 'Feature',
-          assets: {},
+          assets: {
+            metadata: {
+              href: 'https://cmr.earthdata.nasa.gov/search/concepts/1.native',
```

Comment:
Just looking for clarification- is line 23 of `lib/util/app.js` supposed to transform this resulting href into a .xml file?

Reply:
Yup, didn't notice that didn't convert. I've updated the code here.